backend/apps/owasp/migrations/0026_post.py

```
@@ -0,0 +1,30 @@
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('owasp', '0025_remove_event_country_remove_event_postal_code_and_more'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Post',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)